src/app/core/components/generic-mod-list/generic-mod-list.component.ts

```diff
@@ -116,20 +139,24 @@ export default class GenericModListComponent implements AfterViewInit {
         const pageNumbers = Array.from(elements).map(li => parseInt(li.textContent ?? ''));
 
         this.accumulatedModList = Array.from(modList)
-          .map(
-            e =>
-              ({
-                name: e.getElementsByClassName('filebaseFileSubject')[0].getElementsByTagName('span')[0].innerHTML,
-                fileUrl: e.getElementsByTagName('a')[0].href,
-                image: e.getElementsByClassName('filebaseFileIcon')[0]?.getElementsByTagName('img')[0]?.src ?? null,
-                icon: e.getElementsByClassName('filebaseFileIcon')[0]?.getElementsByTagName('span')[0]?.className.split('icon icon128')[1] ?? null,
-                teaser: e.getElementsByClassName('filebaseFileTeaser')[0].innerHTML ?? '',
-                supportedAkiVersion: e.getElementsByClassName('labelList')[0]?.getElementsByClassName('badge label')[0]?.innerHTML ?? '',
-                akiVersionColorCode: e.getElementsByClassName('labelList')[0]?.getElementsByClassName('badge label')[0]?.className,
-                kind: undefined,
-              }) as Mod
-          )
-          .filter(e => this.filterCoreMods(e));
+        .map(
+          e => {
+            const datetimeAttribute = e.querySelector('.filebaseFileMetaData .datetime')?.getAttribute('datetime');
+            const date = datetimeAttribute ? new Date(datetimeAttribute) : undefined;
```

Comment:
See the change detection comment in the template.
It would be totally fine to add the transformed date to the initial fetch of the data and add it there. Line 155 lastUpdate

Reply:
Saving the transformed date now.